stable/metrics-server/README.md

```diff
@@ -28,9 +28,10 @@ Parameter | Description | Default
 `podAnnotations` | Annotations to be added to pods | `{}`
 `priorityClassName` | Pod priority class | `""`
 `readinessProbe` | Container readiness probe | See values.yaml
-`service.annotations` | Annotations to add to the service | `{}`
-`service.port` | Service port to expose | `443`
-`service.type` | Type of service to create | `ClusterIP`
+`service.Annotations` | Annotations to add to the service | `{}`
```

Comment:
Can we change all of these service var names back to camelCase please?

Reply:
I just missed the port one in the test, let's try this first.